File: scripts/clean.mjs
import { rmSync, existsSync } from 'fs';
import { join } from 'path';

const root = join(import.meta.dirname, '..');

const dirsToClean = [
  'apps/web/.next',
  'apps/web/node_modules',
  'apps',
  'packages',
  'node_modules/.cache',
  '.next',
];

for (const dir of dirsToClean) {
  const fullPath = join(root, dir);
  if (existsSync(fullPath)) {
    try {
      rmSync(fullPath, { recursive: true, force: true });
      console.log(`Cleaned: ${dir}`);
    } catch (err) {
      console.log(`Could not clean ${dir}: ${err.message}`);
    }
  } else {
    console.log(`Skipped (not found): ${dir}`);
  }
}

console.log('Done! Project is clean.');

File: scripts/clean.py
import shutil
import os

root = "/vercel/share/v0-project"

dirs_to_remove = [
    os.path.join(root, "apps"),
    os.path.join(root, "packages"),
    os.path.join(root, ".next"),
    os.path.join(root, "node_modules"),
]

for d in dirs_to_remove:
    if os.path.exists(d):
        print(f"Removing {d}...")
        shutil.rmtree(d)
        print(f"  Removed {d}")
    else:
        print(f"  Skipping {d} (not found)")

print("Done! Stale directories cleaned.")
